welcome.md

```diff
@@ -4,10 +4,21 @@ sd_hide_title: true
 
 # Home
 
+<center><img src="https://github.com/pymc-devs/brand/blob/main/logos/1-pymcdevs.png?raw=true" width="75%"/></center>
+
 {doc}`PyMC <pymc:index>` is a probabilistic programming library for Python that allows users to build Bayesian models with a simple Python API and fit them using Markov chain Monte Carlo (MCMC) methods.
-PyMC focuses on usability, flexibility, scalability, and extensibility.
-Along with core model specification and fitting functionality,
-PyMC integrates with {doc}`ArviZ <arviz:index>` for exploratory analysis of the results.
+
+## Features
+PyMC strives to make Bayesian modeling as simple and painless as possible, allowing users to focus on their problem rather than the methods.
+
+Here is what sets it apart:
+
+* **Modern**: Includes state-of-the-art inference algorithms, including MCMC (NUTS) and variational inference (ADVI).
+* **User friendly**: Write your models using friendly Python syntax. [Learn Bayesian modeling](https://www.pymc.io/projects/docs/en/latest/learn.html#) from the many [example notebooks](https://www.pymc.io/projects/examples/en/latest/gallery.html).
+* **Fast**: Uses {doc}`Aesara <aesara:index>` as its computational backend to compile to C and JAX, [run your models on the GPU](https://www.pymc-labs.io/blog-posts/pymc-stan-benchmark/), and benefit from complex graph-optimizations.
+* **Batteries included**: Includes probability distributions, Gaussian processes, ABC, SMC and much more. It integrates nicely with {doc}`ArviZ <arviz:index>` for visualizations and diagnostics.
```

Comment:
Do we want to mention Bambi here like we currently mention Arviz?  If not here, maybe somewhere else?